qiita_db/support_files/patches/python_patches/14.py

```diff
@@ -0,0 +1,66 @@
+# Feberuary 7, 2015
+# This patch recreates all the QIIME mapping files to avoid lower/upper case
+# problems. See https://github.com/biocore/qiita/issues/799
+#
+# heavily based on 7.py
+
+from os.path import basename
+
+from qiita_db.util import get_mountpoint
+from qiita_db.sql_connection import SQLConnectionHandler
+from qiita_db.metadata_template import PrepTemplate
+
+conn_handler = SQLConnectionHandler()
+
+_id, fp_base = get_mountpoint('templates')[0]
+
+sql = "SELECT prep_template_id FROM qiita.prep_template"
+for prep_template_id in conn_handler.execute_fetchall(sql):
+
+    prep_template_id = prep_template_id[0]
+    pt = PrepTemplate(prep_template_id)
+    fps = pt.get_filepaths()
+
+    # get the QIIME mapping file, note that the way to figure out what is and
+    # what's not a qiime mapping file is to check for the existance of the
+    # word qiime in the basename of the file path, hacky but that's the way
+    # it is being done in qiita_pet/uimodules/raw_data_tab.py
```

Comment:
Is there perhaps a better way to do this though? It would be nice to have a magic number but that would require difficult-in-practice changes to qiime. What about extending the filename to "_qiime_mapping_file_" to be even more explicit?